core/main/src/service/context_manager.rs

```diff
@@ -75,6 +75,34 @@ impl ContextManager {
             warn!("No processor to set TimeZoneChanged status listener")
         }
 
+        // Setup the DisplayConnectionChanged status listener
+        debug!("Subscribing for change in Display Connection");
+        if ps
+            .get_client()
+            .send_extn_request(DeviceEventRequest {
+                event: DeviceEvent::InputChanged,
```

Comment:
I'm not sure we should be using `DeviceEvent::InputChanged` at all with any of this, I think this refers to HDMI inputs whereas HDCP relates to when the OUTPUT device (TV) is changed (e.g. device connection has changed).

Reply:
...in other words it seems like we need to rename `DeviceEvent::InputChanged` to `DeviceEvent::HdcpChanged`. But note the contract name should also be changed (`EventAdjective::Input` --> `EventAdjective::Hdcp`) in code and manifests.